maat-court-data-api/src/main/java/gov/uk/courtdata/repository/RepOrderRepository.java

```diff
@@ -1,12 +1,29 @@
 package gov.uk.courtdata.repository;
 
 import gov.uk.courtdata.entity.RepOrderEntity;
+import gov.uk.courtdata.reporder.projection.IOJAssessorDetails;
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.data.jpa.repository.JpaSpecificationExecutor;
+import org.springframework.data.jpa.repository.Query;
 import org.springframework.stereotype.Repository;
 
 
 @Repository
 public interface RepOrderRepository extends JpaRepository<RepOrderEntity, Integer>, JpaSpecificationExecutor<RepOrderEntity> {
 
+    @Query(value = """
```

Comment:
Why is there three punctuation marks around the value?

Reply:
I'm guessing it's a multi-line string?

Reply:
yes, it is a feature that came with Java 15, I felt it made the query significantly easier to read compared to the alternatives.

Like:
```
    @Query(value = "   SELECT" +
                   "       NVL(INITCAP(U.FIRST_NAME" +
                   "                   || ' '" +
                   "                   || U.SURNAME)," +
                   "           R.USER_CREATED) name," +
                   "           R.USER_CREATED userName" +
                   "   FROM" +
                   "       REP_ORDERS R" +
                   "       LEFT JOIN USERS U ON (U.USER_NAME = R.USER_CREATED)" +
                   "   WHERE" +
                   "       ID = :repOrderId",
```
Or:
```
    @Query(value = " SELECT NVL(INITCAP(U.FIRST_NAME || ' ' || U.SURNAME), R.USER_CREATED) name, R.USER_CREATED userName FROM REP_ORDERS R LEFT JOIN USERS U ON (U.USER_NAME = R.USER_CREATED) WHERE ID = :repOrderId",
            nativeQuery = true)
```

https://www.baeldung.com/java-text-blocks